root/usr/share/nethesis/NethServer/Language/en/NethServer_Module_Mail_Domain.php

```diff
@@ -2,8 +2,6 @@
 
 $L['Create a new domain'] = 'Create a new domain';
 $L['Delete_label'] = 'Delete';
-$L['DisclaimerStatus_label'] = 'Append a legal note to sent messages';
```

Comment:
We need this line for Transifex: if two alternative packages exist the same string catalog file have to support both of them